### PR TITLE
Milestone 3: Interactive form

### PR DESCRIPTION
**Pull request by SotoArmando**
Alrighty! The foundation is built. Here comes the real job. In this milestone, we'll connect the CREATE_BOOK and REMOVE_BOOK actions with the React component and make our page dynamic.

### CODE REVIEW

File: src/Containers/bookslist.js [hunk 1]
```diff
@@ -4,47 +4,73 @@ import { connect } from 'react-redux';
 import Msgtitle from '../Components/msgtitle';
 import Booksform from './booksform';
 import Book from './book';
+import Categoriesform from './categoriesform';
 
 function Bookslist(props) {
-  const { bookstore } = props;
+  const {
+    bookstore, filtered, REMOVE_BOOK, UPDATE_BOOK,
```
Comment (azdrenymeri) [inline src/Containers/bookslist.js:11]: Why these two functions `REMOVE_BOOK`,`UPDATE_BOOK` are all upper case :thinking:

File: src/Containers/book.js [hunk 2]
```diff
@@ -1,46 +1,135 @@
...
+    const { deleteindex, handleRemoveBook } = this.props;
+    return (
+      <div className="notshown Book">
+        <div className="col">
+          <span className="fw-700">{category}</span>
+          <span>{title}</span>
+          <span>{author}</span>
+          <span className="sidemenu">
+            <span role="button" tabIndex={0} onClick={() => { handleRemoveBook(deleteindex); }}>Remove</span>
+            <span role="button" tabIndex={0} onClick={this.handleEditmode}>Edit</span>
```
Comment (azdrenymeri) [inline src/Containers/book.js:72]: `OPTIONAL` I would change these to buttons or apply a `noselect` CSS property so when hovering over these two elements it shows the arrow cursor and not the text selector delimeter.

**Review by azdrenymeri — APPROVED** [review]
## Hello @SotoArmando :wave: 
### Great job on this milestone :+1: 
#### All requirements are met :heavy_check_mark:

File: src/Containers/bookslist.js (hunk 1 above)
Comment (SotoArmando) [reply src/Containers/bookslist.js:11]: sometimes happens that i do no brain things ... like just write again same doc references xd.

File: src/Containers/book.js (hunk 2 above)
Comment (SotoArmando) [reply src/Containers/book.js:72]: thanks 👍

**Pull request merged**